### PR TITLE
fix(clerk-js): User/org profile spacing adjustments

### PR DESCRIPTION
**Pull request by desiprisg**
## Description

Before:
Screenshot 2024-02-01 at 23 03 09

After:
Screenshot 2024-02-01 at 23 03 42

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by octoper** [comment]
!preview

File: packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ const DeviceAccordion = ({ session }: { session: SessionWithActivitiesResource }
           sx={t => ({
             alignItems: 'flex-start',
             padding: `${t.space.$2} ${t.space.$4}`,
+            marginLeft: `-${t.space.$4}`,
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx:64]: Why do we use negative margins?

**Comment by anagstef** [comment]
@desiprisg can we have `before` and `after` screenshots?

File: packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx (hunk 1 above)
Comment (desiprisg) [reply packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx:64]: We need to position the button to the left to align text with the items in the list

**Comment by anagstef** [comment]
!preview

**Comment by anagstef** [comment]
!preview

**Review by anagstef — APPROVED** [review]

**Pull request merged**